article/app/controllers/ArticleController.scala

```diff
@@ -109,9 +109,11 @@ class ArticleController(
         Future.successful(common.renderJson(getJson(article), article))
       case EmailFormat =>
         Future.successful(common.renderEmail(ArticleEmailHtmlPage.html(article), article))
+      case HtmlFormat | AmpFormat if tier == PressedArticle =>
+        servePressedPage(path)
       case AmpFormat if isAmpSupported =>
         remoteRenderer.getAMPArticle(ws, article, blocks, pageType)
-      case HtmlFormat | AmpFormat if tier == RemoteRender =>
+      case HtmlFormat | AmpFormat if tier == DotcomRendering =>
```

Comment:
This rename doesn't sync with the function `remoteRenderer`. Can we either lean into it and make the change in all places we use 'remote' or pull back and keep the remote nomenclature. Maybe in a different PR?

Reply:
Makes sense, we reverted the renaming for now.